ketcher-autotests/tests/Macromolecule-editor/Macro-Micro-Switcher/macro-micro-switcher.spec.ts

```diff
@@ -0,0 +1,439 @@
+import {
+  turnOnMacromoleculesEditor,
+  turnOnMicromoleculesEditor,
+} from '@utils/macromolecules';
+import { Page, test } from '@playwright/test';
+import {
+  FILE_TEST_DATA,
+  TopPanelButton,
+  clickInTheMiddleOfTheScreen,
+  openFileAndAddToCanvas,
+  pressButton,
+  selectTopPanelButton,
+  takeEditorScreenshot,
+  takeMonomerLibraryScreenshot,
+  takePageScreenshot,
+  waitForLoad,
+  waitForPageInit,
+  waitForRender,
+} from '@utils';
+
+async function pasteFromClipboard(page: Page, fileFormats: string) {
+  await selectTopPanelButton(TopPanelButton.Open, page);
+  await page.getByText('Paste from clipboard').click();
+  await page.getByRole('dialog').getByRole('textbox').fill(fileFormats);
+  await waitForLoad(page, async () => {
+    await pressButton(page, 'Add to Canvas');
+  });
+}
+
+async function addToFavoritesMonomers(page: Page) {
+  await page.getByTestId('Bal___beta-Alanine').getByText('★').click();
```

Comment:
not worth it to create a function? TestID-> text ->click
just a question, feel free to ignore